lib/metrics.js

```diff
@@ -28,10 +28,22 @@ export class Metrics {
    * - Writes `jobs-completed` to InfluxDB
    * - Merges metrics from all modules
    * @param {String} moduleName
-   * @param {MetricsEvent} metrics
+   * @param {Partial<MetricsEvent>} metrics
    */
   submit (moduleName, metrics) {
+    /** @type {MetricsEvent} */
+    const resolvedMetrics = {
```

Comment:
```suggestion
    const mergedMetrics = {
```

Reply:
I cannot use the name `mergedMetrics` because we already have a variable with that name declared below.

```js
    const mergedMetrics = {
      totalJobsCompleted: 0,
      rewardsScheduledForAddress: 0n
    }
    for (const [, metrics] of this.moduleMetrics) {
      mergedMetrics.totalJobsCompleted += metrics.totalJobsCompleted
      // Merging rewards metrics should be revisited as more modules start
      // paying rewards
      mergedMetrics.rewardsScheduledForAddress +=
        metrics.rewardsScheduledForAddress
    }
```

Reply:
I am going to land this PR as it is now; feel free to open a follow-up PR to rename `resolvedMetrics` to something better.